Package.swift

```diff
@@ -149,7 +149,7 @@ let package = Package(
       url: "https://github.com/google/GoogleAppMeasurement.git",
       // Please keep the version specification aligned with
       // scripts/setup_spm_test_app_measurement.sh.
-      .exact("8.3.1")
+      .exact("8.7.0")
```

Comment:
nit: update the above comment now that `scripts/setup_spm_test_app_measurement.sh` is being deleted

Reply:
Good catch! Thanks.